site2/docs/schema-understand.md

```diff
@@ -122,6 +122,11 @@ Pulsar gets the schema definition from the predefined `struct` using an Avro lib
 1. Create the _User_ class to define the messages sent to Pulsar topics.
 
    ```java
+   # If you use Lombok
+   @Builder
+   @AllArgsConstructor
+   @NoArgsConstructor
```

Comment:
We can remove line 133 to 137 since lombok will generate the constructors.

Reply:
Thanks I just changed it a i though it was.

Removed.